x/feegrant/exported/external.go

```diff
@@ -19,3 +21,17 @@ type AccountKeeper interface {
 	GetAccount(ctx sdk.Context, addr sdk.AccAddress) auth.Account
 	SetAccount(ctx sdk.Context, acc auth.Account)
 }
+
+type MsgGrantFeeAllowance interface {
+	sdk.Msg
+
+	GetFeeGrant() *FeeAllowance
```

Comment:
The original message:

```go
type MsgGrantFeeAllowance struct {
	Granter   sdk.AccAddress        `json:"granter" yaml:"granter"`
	Grantee   sdk.AccAddress        `json:"grantee" yaml:"grantee"`
	Allowance exported.FeeAllowance `json:"allowance" yaml:"allowance"`
}
```

So shouldn't this interface be:

```go
type MsgGrantFeeAllowance interface {
  sdk.Msg

  GetGranter() sdk.AccAddress
  GetGrantee() sdk.AccAddress
  GetFeeAllowance() FeeAllowance
}
```

?

Reply:
Edit: I see we have `MsgGrantFeeAllowance` defined in the proto schema. Do we even need these types then?

Reply:
On further thought, I think these both can be removed entirely. Will change